tests/test_interop_scenarios.rs

```diff
@@ -0,0 +1,263 @@
+use openmls::prelude::*;
+use std::convert::TryFrom;
+use test_macros::ctest;
+
+mod utils;
+
+use utils::managed_utils::*;
+
+// The following tests are test corresponding the the interop test scenarios
+// detailed here:
+// https://github.com/mlswg/mls-implementations/blob/master/test-scenarios.md
+// The tests are conducted for every available ciphersuite, but currently only
+// using BasicCredentials. We can change the test setup once #134 is fixed.
+
+fn default_managed_group_config() -> ManagedGroupConfig {
+    let handshake_message_format = HandshakeMessageFormat::Plaintext;
+    let update_policy = UpdatePolicy::default();
+    let callbacks = default_callbacks::default_callbacks();
+    ManagedGroupConfig::new(handshake_message_format, update_policy, 10, callbacks)
+}
+
+/// # 1:1 join
```

Comment:
You can't do doc comments here. Just make all of them normal ones.

Reply:
Done!